packages/subproviders/src/subproviders/injected_web3.ts

```diff
@@ -42,8 +41,9 @@ export class InjectedWeb3Subprovider {
         }
     }
     // Required to implement this method despite not needing it for this subprovider
+    // This type is Web3ProviderEngine, but there is no need to import this for a noop.
```

Comment:
// The `engine` argument type should be `Web3ProviderEngine` but we've decided to keep it as type `any` to // remove the providerEngine dependency given that this method is a noop anyways